src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/daos/ContentDao.kt

```diff
@@ -67,6 +67,7 @@ class ContentDao(
                         CONTENT_ENTITY.COLLECTION_FK,
                         CONTENT_ENTITY.SORT,
                         CONTENT_ENTITY.START,
+                        CONTENT_ENTITY.END,
```

Comment:
Do we need `end`?

Reply:
I see that in your `MarkdownProjectReader` implementation, you currently set both `start` and `end` to the `fileId`, which apparently will be 0 for the intro.md files. We had talked about using both `start` and `end` before, but I don't think there's a reason to if the only entries that will use it are contents that are resources for chapters (`start` = 0 seems to work just fine.)

Reply:
I had omitted it because it could be derived, and could just be calculated on mapping entities to our app data classes; but it's also probably not a big deal and if you really want to add it, it really wouldn't be much of a cost.